trigger/task.go

```diff
@@ -355,7 +355,7 @@ func (t *taskTriggers) generate(sub *event.Subscription, pastTenseOverride, test
 					return nil, errors.Wrapf(err, "error getting execution task")
 				}
 				if executionTask.Details.Status == evergreen.TaskFailed {
-					if executionTask.Details.Description == evergreen.TaskDescriptionStranded {
+					if executionTask.Details.Description == evergreen.TaskDescriptionStranded || executionTask.Details.Description == evergreen.TaskSetupFailed {
```

Comment:
We do not set setup-failed as the description of the task details, in this way it is unlike the taskDescriptionStranded property.  It is set as the task's display status, see here : https://evergreen.mongodb.com/api/rest/v2/tasks/mongodb_mongo_v5.1_enterprise_windows_powercycle_replication_smalloplog_0_enterprise_windows_b21f0146559bab88eb51d73dff6f896cf8bc5a9c_21_11_17_21_10_33?execution=1

Reply:
You can check the codebase for instances where we apply evergreen.TaskSetupFailed to a task.